cl/persistence/state/historical_states_reader/historical_states_reader.go

```diff
@@ -478,12 +476,11 @@ func (r *HistoricalStatesReader) reconstructBalances(tx kv.Tx, slot uint64, diff
 		if base_encoding.Decode64FromBytes4(k) > slot {
 			return nil, fmt.Errorf("diff not found for slot %d", slot)
 		}
-		s := time.Now()
+		fmt.Println("AX")
```

Comment:
I think this is only for debugging, if so let's remove before merge the PR.

Reply:
Yeah, I removed it, I noticed I introduced a bug with my latest other PR